### PR TITLE
Nuzhy/85537/refactor partners page

### PR DESCRIPTION
**Pull request by Nuzhy-Deriv**
Changes:

-   ...

## Type of change

-   [ ] Bug fix
-   [ ] New feature
-   [ ] Update feature
-   [x] Refactor code
-   [ ] Translation to code
-   [ ] Translation to crowdin
-   [ ] Script configuration
-   [ ] Improve performance
-   [ ] Style only
-   [ ] Dependency update
-   [ ] Documentation update
-   [ ] Release

### CODE REVIEW

File: crowdin/messages.json [hunk 1]
```diff
@@ -29,6 +29,7 @@
   "31715193": "High Ticks/Low Ticks, Asians, Reset Call/Reset Put, Digits, and Only Ups/Only Downs are available exclusively on synthetic indices.",
```
[the remark this answers is not in the document]
Comment (Nuzhy-Deriv) [reply crowdin/messages.json:29]: sure

**Comment by Nuzhy-Deriv** [comment]
@niloofar-deriv I have merged the base branch and resolved the conflicts, plus have made changes as you suggested. Please help to review. Thank you

File: src/pages/partners/affiliate-ib/index.tsx [hunk 2]
```diff
@@ -123,47 +124,42 @@ const StyledText = styled(Text)`
...
 
 const items: ItemsType = [
-    { title: '60K+', subtitle: <Localize translate_text="members" /> },
-    { title: '$47M+', subtitle: <Localize translate_text="paid out since inception" /> },
-    { title: '190+', subtitle: <Localize translate_text="countries" /> },
-    { title: '1M+', subtitle: <Localize translate_text="clients" /> },
+    { title: '_t_60K+_t_', subtitle: '_t_members_t_' },
+    { title: '_t_$47M+_t_', subtitle: '_t_paid out since inception_t_' },
+    { title: '_t_190+_t_', subtitle: '_t_countries_t_' },
+    { title: '_t_1M+_t_', subtitle: '_t_clients_t_' },
```
Comment (niloofar-deriv) [inline src/pages/partners/affiliate-ib/index.tsx:130]: same

Comment (Nuzhy-Deriv) [reply src/pages/partners/affiliate-ib/index.tsx]: I have made the changes as suggested. Thanks

**Review by niloofar-deriv — APPROVED** [review]

**Pull request merged**